syntax = "proto3";

package auth_service;

service AuthService
{
	rpc LoginProcedure(AuthRequest) returns (AuthResponse);
	rpc SignupProcedure(AuthRequest) returns (AuthResponse);
}

message AuthRequest
{
	string username = 1;
	string password = 2;
}
message AuthResponse
{
	uint32 client_id = 1;
}